dockerfiles/nginx/proxito.conf

```diff
@@ -66,6 +66,8 @@ server {
         add_header Cache-Tag $cache_tag always;
         set $referrer_policy $upstream_http_referrer_policy;
         add_header Referrer-Policy $referrer_policy always;
+        set $permissions_policy $upstream_http_permissions_policy;
+        add_header Permissions-Policy $permissions_policy always;
```

Comment:
We should probably add the same in `web.conf` to make it work int the same way on the dashboard. It's not _required_ sine this is locally, but I think it makes sense to keep consistency with prod.

Reply:
I don't think I understand this proposal. `web.conf` ([here](https://github.com/readthedocs/readthedocs.org/blob/master/dockerfiles/nginx/web.conf)) doesn't have any nginx configuration to include headers -- either here or in the private operations repo.

Reply:
Oh, yeah. You are right! I got confused about what was the case were this was needed. Thanks!